source/jackett/jackett_service.py

```diff
@@ -212,8 +213,21 @@ def __get_torrent_links_from_xml(self, media, xml_content):
 
             infoHash = item.find('.//torznab:attr[@name="infohash"]',
                                  namespaces={'torznab': 'http://torznab.com/schemas/2015/feed'})
-            result.infoHash = infoHash.attrib['value'] if infoHash is not None else None
+            result.info_hash = infoHash.attrib['value'] if infoHash is not None else None
 
             result_list.append(result)
 
         return result_list
+    
+    def __post_process_results(self, results, media):
+        for result in results:
+            result.language = detection.detect_language(result.title)
+            result.quality = detection.detect_quality(result.title)
+            result.quality_spec = detection.detect_quality_spec(result.title)
+            result.type = media.type
+
+            if isinstance(media, Series):
```

Comment:
media.type should also work

Reply:
For some reason I prefer the isinstace method. I'll keep it this way for now

Reply:
But then let's remove the type attribute from the media class